.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.24-bookworm
```

Comment:
Generally, you'll want to use [`mcr.microsoft.com/vscode/devcontainers/go`](https://github.com/devcontainers/images/tree/main/src/go). So that we have all the necessary essentials to run IDE inside the container